moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -103,6 +110,17 @@ const useStyles = makeStyles(theme => ({
   indicatorColor: {
     backgroundColor: theme.palette.primary.light,
   },
+  followDiv: {
+    float: "right",
+  },
+  unfollowIcon: {
+    color: theme.palette.primary.main,
+    fontSize: "2rem",
+  },
+  followIcon: {
+    color: theme.palette.text.secondary,
+    fontSize: "2rem",
+  },
```

Comment:
These icons turned out really great. 💯 I love how they look - they communicate what the feature is and how to use it with so little - it's very elegant.

Reply:
thanks, @frankhereford! I'm starting down here and working my way up